Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -3357,38 +3357,19 @@ public static partial class PythonOps {
             return DoubleOps.Compare(self, other);
         }
 
-        [Obsolete("Use Bytes(IList<byte>) instead.")]
+        [Obsolete("Use Bytes(IList<byte>) or Bytes(ReadOnlySpan<byte>) instead.")]
         public static Bytes MakeBytes(byte[] bytes) {
-            return new Bytes(bytes);
+            return new Bytes(bytes.AsSpan());
```

Comment:
Why add `AsSpan`? I think this should probably have been `Bytes.Make` anyway.

Reply:
Yes, this should have been `Bytes.Make` here. And maybe everywhere else where `byte[]` is being passes to `Bytes` constructor, instead of casting to `IList` or `Span`, use `Bytes.Make` or `PythonOps.MakeBytes`, since `Bytes.Make` is internal. I haven't done it because `PythonOps.MakeBytes` is marked obsolete and is being used in only one place, but maybe it can find a new purpose?

Reply:
I think I was the one to mark `PythonOps.MakeBytes` as obsolete since I didn't want it to be used to create `Bytes` objects. Not sure I really want to (for the moment) have a public method to create `Bytes` from an array.

Reply:
By the way, do you know how to create a `bytes[]` instance in IronPython (that is `System.Array` with elements of `System.Byte`)?

Reply:
`System.Array[System.Byte]([1, 2, 3])`

Reply:
I see. I was trying to create an empty (zero-filled) array of a given size:
```
>>> System.Array[System.Byte](1000)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: expected object with __len__ function, got int
```

Reply:
Presumably we could add an `int` overload to `ArrayOps` to allow this?

Reply:
I think it would be good thing to implement. I would bring .NET arrays on par with `bytes` and `bytearray`, which both accept an int to specify a length of a zero-filled array.

Reply:
I've found it:
```
System.Array.CreateInstance(System.Byte, 1000)
```
Nevertheless, I will submit a PR to support a more Pythonic syntax as well.